api/Avalonia.nupkg.xml

```diff
@@ -1,6 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
```

Comment:
This needs to be removed: it's supressing the binary breaking change introduced in this PR.

Reply:
I pushed an update with these lines removed and created a new function named CreateEditDrawingContext() that creates a DrawingContext to editing the already existing image data instead of clearing it. Removed the optional parameter from CreateDrawingContext().

Reply:
```csharp
void CreateDrawingContext() =>
CreateDrawingContext(true);

void CreateDrawingContext(boom clear)
{
 // your code
}

```

Reply:
Switched to this pattern, thanks!